tests/Integration/UseCases/CancelMembershipApplication/CancelMembershipApplicationUseCaseTest.php

```diff
@@ -191,13 +192,19 @@ private function whenCancelApplicationRequestIsSentByAdmin( CancelMembershipAppl
 		$useCase->cancelApplication( new CancellationRequest( $applicationId, self::AUTH_USER_NAME ) );
 	}
 
+	/**
+	 * @return array{FakeApplicationRepository,MembershipApplication}
```

Comment:
```suggestion
	 * @return array{0:FakeApplicationRepository,1:MembershipApplication}
```
It's a pity that we'd have to add "clutter" to to the definition to make Scrutinizer happy. Your syntax is perfectly valid and reads better than the positional one.